### PR TITLE
refactor(rxNotify): move components/rxNotify to elements/rxNotify

### PR DESCRIPTION
**Pull request by khaziwallis**
JIRA:

### LGTMs
- [ ] Dev LGTM
- [ ] Dev LGTM
- [ ] Design LGTM
- [ ] QE LGTM

Navigation 
screen shot 2016-07-18 at 4 46 03 pm

Search
screen shot 2016-07-18 at 5 54 17 pm
screen shot 2016-07-18 at 5 54 27 pm
screen shot 2016-07-18 at 5 54 40 pm

### CODE REVIEW

**Comment by Droogans** [comment]
@khaziwallis You can't push a pull request up from your fork located at https://github.com/khaziwallis/encore-ui, since our CI tests won't run against pull requests originating from forks.

If you are unable to push this branch to https://github.com/rackerlabs/encore-ui, please leave a comment here stating that and I'll get the PR going for you.

**Comment by glynnis** [comment]
@khaziwallis When submitting a PR, please paste the link to the relevant JIRA story. You should also include before and after screen shots showing what visual changes resulted from the work you did. See https://github.com/rackerlabs/encore-ui/blob/master/CONTRIBUTING.md#design-review-of-pull-requests for more detail.

I see that you've included some screen shots here, which is helpful! But there are more changes that resulted from your PR than you've included in your screen shots. A good example of before/after screen shots for this kind of PR can be found here: https://github.com/rackerlabs/encore-ui/pull/1728#issuecomment-228388957

Thanks!

File: grunt-tasks/examples.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ module.exports = function (grunt) {
         // }
         for (key in metadata) {
             if (examples[key]) {
-                throw new Error(`Examples for '${key}' already defined.`);
+                throw new Error("Examples for '${key}' already defined.");
```
Comment (100stacks) [inline grunt-tasks/examples.js]: @khaziwallis, we are using [ES6 template strings](https://hacks.mozilla.org/2015/05/es6-in-depth-template-strings-2/) here.  Please revert your changes.

File: grunt-tasks/examples.js [hunk 2]
```diff
@@ -86,7 +86,7 @@ module.exports = function (grunt) {
                         example.less = fileContent;
                         break;
                     default:
-                        throw new Error(`Unknown file extension: ${ext}`);
+                        throw new Error("Unknown file extension: '${ext}'");
```
Comment (100stacks) [inline grunt-tasks/examples.js]: @khaziwallis, we are using [ES6 template strings](https://hacks.mozilla.org/2015/05/es6-in-depth-template-strings-2/) here.  Please revert your changes.

**Comment by 100stacks** [comment]
@khaziwallis, as @Droogans mentioned the reason above....you will have to close this PR and open a new PR with a branch off `master` to trigger the CI process.

**Comment by 100stacks** [comment]
@khaziwallis, in addition to the great feedback provided above, please see below:

0. **First** commit is **only** for moving files from one location to another. Use `git mv` for this step to move files.
0. **Second** commit should contain file edits or file removals.  Use `git rm` to remove a file.
0. To utilize the new `rxExample` construct you need a new file in a new  `elements/elementName/docs` folder called: `elementName.docs.html`   
0.  `elementName.docs.html` will looks something like this:
    ```html
    <p>
       The tooltips component provides styles to raw HTML elements and custom directive templates.
    </p>

     <rx-example name="tooltips.simple"></rx-example>
    ```

    where `tooltips.simple` above is `elementName.simple` which points to  a atomic demo and/or
    feature.
    `rxExample` looks in `elements/elementName/docs/examples` folder for `elementName.simple.html`
    and (if applicable) `elementName.simple.js` files.
0. Please see PR #1728 as an example of how to do this: https://github.com/rackerlabs/encore-ui/pull/1728/files#diff-1fbdb3a88ad53fc359b7cf5e22d3cec9  

Each demo example will be contained in a separate atomic `rxExample` construct.

**Comment by 100stacks** [comment]
@khaziwallis, closing this PR based on above feedback.

**Pull request closed without merging**